src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -276,6 +301,32 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /** initialize courses attribute by retreiving recently accessed courses from the server */
+    getRecentlyAccessedCourses() {
+        this.courseService.findAllForDashboard().subscribe({
+            next: (res: HttpResponse<CoursesForDashboardDTO>) => {
+                if (res.body) {
+                    const courses: Course[] = [];
+                    for (const courseDto of res.body.courses) {
+                        courses.push(courseDto.course);
+                    }
+                    this.courses = courses.sort((a, b) => (a.title ?? '').localeCompare(b.title ?? ''));
+                    if (this.courses.length > 3) {
```

Comment:
This condition is quite simple, but I do still think that it would be cleaner not to introduce magic numbers. 
```suggestion
                    if (this.courses.length > MIN_COURSES_TO_DISPLAY_QUICKSELECTION) {
```

Reply:
agreed